styles.css

```diff
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: #07535b;
+}
+
+.header-toolbar {
+  display: flex;
+  flex-direction: row;
+  min-height: 4rem;
+  align-items: center;
+  padding: 15px;
+  color: white;
+  opacity: 1;
```

Comment:
this is also useless as the default opacity is 1 no need to add except you want to play with opacity